src/dev.ts

```diff
@@ -14,7 +15,7 @@ export function shutdownWatcher() {
 }
 
 export async function startDevWatcher() {
-  shutdownWatcher()
+  exitHook(shutdownWatcher)
```

Comment:
This line was called to ensure closing **previous** dev watcher when configuration changes. 
The hook is better to be applied in https://github.com/antfu/postcss-windicss/blob/a09a56e3e2b3b3d736ebeab1165679a6716ff708/src/index.ts#L22-L23
with `isDev` check.

Reply:
In line 6, the variable watcher is defined as undefined, so that the function call in line 17 is always ineffective. Because the condition in line 9 is always false.

I think, we should close the watcher if the node process terminated.

Reply:
Nope, `startDevWatcher` could be called multiple times.

> I think, we should close the watcher if the node process terminated.

Yeah, I agree, and that's what I mean. The exit hook only needs to be registered one-time.

Reply:
Okay, I didn't know that. No problem ...